RecoParticleFlow/PFClusterProducer/plugins/SimMappers/ComputeClusterTime.h

```diff
@@ -0,0 +1,122 @@
+// user include files
+#include <vector>
+
+
+// functions to select the hits to compute the time of a given cluster
+// start with the only hits with timing information
+// average among the hits contained in the chosen time interval
+
+// N.B. time is corrected wrt vtx-calorimeter distance 
+// with straight line and light speed hypothesis
+// for charged tracks or heavy particles (longer track length or beta < 1) 
+// need to correct the offset at analysis level
+
+
+
+//time-interval based on the smallest one containing a minimum fraction of hits
+float highestDensityFraction(std::vector<float>& hitTimes, float fractionToKeep=0.68 /*fraction between 0 and 1*/, float timeWidthBy=0.5){
```

Comment:
this function isn't used anywhere?

Reply:
No I will leave it commented out, as reference for future developments, and add a comment line about this.